docs/framework/wcf/feature-details/partial-trust-feature-compatibility.md

```diff
@@ -132,14 +132,14 @@ manager: "erikre"
 -   CardSpace  
   
 -   <xref:System.IO.Log>  
-  
--   <xref:System.ServiceModel.Internal.TransactionBridge>  
+<!--
+-   System.ServiceModel.Internal.TransactionBridge  -->
   
  The following members of the <xref:System.Diagnostics.TraceOptions> enumeration should not be specified:  
   
 -   <xref:System.Diagnostics.TraceOptions>  
   
--   <xref:System.Diagnostics.TraceOptions.ProcessID>  
+-   <xref:System.Diagnostics.TraceOptions.ProcessId>  
```

Comment:
Glad you flagged this one. This wasn't the only prob here. `CallStack` should be the other one. https://msdn.microsoft.com/en-us/library/bb412186(v=vs.110).aspx#Tracing

I'm going to add the MSDN link for `TransactionBridge` temporarily. That one is listed in the Migration Project notes to investigate further.